multiplatform-movies/android/src/main/java/com/novoda/movies/gallery/GalleryActivity.kt

```diff
@@ -25,10 +35,12 @@ class GalleryActivity : AppCompatActivity(), GalleryPresenter.View {
     }
 
     override fun renderError(message: String?) {
-        label.text = message ?: getString(R.string.gallery_error)
+        Toast.makeText(this, message, Toast.LENGTH_LONG).show()
     }
 
     override fun render(gallery: Gallery) {
-        label.text = resources.getString(R.string.gallery_items, gallery.moviePosters.size)
+        galleryAdapter.updateWith(gallery)
     }
+
+    private fun moviePosterWidthInPixels(): Int = resources.displayMetrics.widthPixels / NO_OF_COLUMNS
```

Comment:
The result is truncated (fractional part thrown away) and not rounded to the closest integer when dividing two ints.
Might be fine here because the number will be high but if we need something pixel perfect we probably need to have a float somewhere and then round the result.